src/Controller/Api/InviteController.php

```diff
@@ -76,6 +76,10 @@ public function createInvite(
             );
         }
 
+        if ($inviteDTO->getPermissionType() == PermissionVoter::OWNER) {
```

Comment:
@SebastianStorz please use triple equal signs (===) as the comparison operator in php, otherwise you might get unexpected results